### ISSUE TITLE
MVT keep Z

### ISSUE THREAD

**Issue by Radik24** [CrunchyData/pg_tileserv#146]
Hi,

I tried pg_tileserv on some 3D geometries in PostGIS. I have 3D-ready client and all the geometries are without Z. Is it possible to return tiles also with Z coordinate?

I found https://research.geodan.nl/how-to-make-a-vectortile-basemap article and there is something about keeping Z.

Thanks!

**Comment by Radik24** [comment]
Thank you! Will think about suitable format.

**Comment by pramsey** [comment]
Format problem is [already solved](https://cesium.com/why-cesium/3d-tiles/), it is generating the things in PostGIS which is not.

**Comment by Radik24** [comment]
Yeah I know it. I have many static cesium / i3s, ... tilesets. Now I must generate it somehow from postgis :)
